### DISCUSSION TITLE
Digital Dash Pod Open Source?

### DISCUSSION THREAD

**Discussion by gruntygap** [KaiserEngineering/digitaldash#8] [Q&A]
To the team at Kaiser Engineering, or someone else who knows.

Does this open source project have all the components needed to recreate a unit? I have been unable to find an STL or pod cluster model within these Repos. Additionally, it seems all the links for the discord community have expired?

Is the open source nature of this project code only?

Thanks!

**Comment by MattKai45** [comment]
Only the code is open source in order to protect our product from being "copied". But feel free to reach out on our Discord (https://discord.gg/pJYAEuF4TQ). I may be able to assist you depending on what you are trying to do.
